src/ser/mod.rs

```diff
@@ -476,3 +476,38 @@ impl Error {
             "top-level serializer supports only maps and structs".into())
     }
 }
+
+#[cfg(test)]
+mod test {
+
+    use super::*;
```

Comment:
Sorry, can't get this to work. I get `unresolved import `serde_urlencoded::to_string`. When moving to this to a dedicated tests folder it does work. I think that's a better solution anyway because it's actually an integration test, right? :-)